lib/ui/profile/components/dao_widget.dart

```diff
@@ -13,54 +15,71 @@ class DaoWidget extends StatelessWidget {
-                  //   future: fetchSVGFromIPFS('QmV3KmaoqCCXuCDvHzYWS9Jg3RfjrDTQSXK1e7453qfSRS'),
-                  //   builder: (context, snapshot) {
-                  //     if (snapshot.connectionState == ConnectionState.waiting) {
-                  //       return CircularProgressIndicator();
-                  //     } else if (snapshot.hasError) {
-                  //       return Text('Error: ${snapshot.error}');
-                  //     } else {
-                  //       return SvgPicture.string(snapshot.data!);
-                  //     }
-                  //   },
-                  // ),
-                  visualDensity: VisualDensity.compact,
-                  title: Text(dao.settingsDaoTitle, style: context.hyphaTextTheme.smallTitles),
-                ),
-                // const SizedBox(height: 14),
-                // const HyphaDivider(),
-                // const SizedBox(height: 20),
-              ],
+    return HyphaCard(
+      child: InkWell(
+        borderRadius: BorderRadius.circular(16),
+        onTap: () {
+          /// Navigate to Hypha URL
+        },
+        child: Stack(
+          children: [
+            const Positioned(
+              right: 12,
+              top: 12,
+              child: Icon(Icons.navigate_next),
             ),
-          ),
+            Padding(
+              padding: const EdgeInsets.all(24),
+              child: Column(
+                mainAxisAlignment: MainAxisAlignment.start,
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: [
+                  Row(
+                    mainAxisAlignment: MainAxisAlignment.start,
+                    children: [
+                      const Icon(Icons.ac_unit, size: 48),
+                      const SizedBox(width: 8),
+                      Column(
+                        crossAxisAlignment: CrossAxisAlignment.start,
+                        children: [
+                          Text(dao.settingsDaoTitle, style: context.hyphaTextTheme.smallTitles),
+                          Text(
+                            'dao.hypha.earth/${dao.settingsDaoUrl}',
+                            style: context.hyphaTextTheme.reducedTitles.copyWith(color: HyphaColors.primaryBlu),
+                          ),
+                        ],
+                      )
+                    ],
+                  ),
+                  const SizedBox(height: 12),
+                  const HyphaDivider(),
+                  const SizedBox(height: 12),
+                  Row(
+                    crossAxisAlignment: CrossAxisAlignment.center,
+                    mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                    children: [
+                      Row(
+                        children: [
+                          const Icon(Icons.calendar_month),
+                          const SizedBox(width: 4),
+                          Text(
+                            'The date here',
```

Comment:
Should this say "the date here"?

Reply:
We dont have this field in model response. 

Is this data available for every dao? 

Do you have an account with multiple daos i can use?

Reply:
nvm Leo will let me use hers

Reply:
![image](https://github.com/hypha-dao/hypha_wallet/assets/8041901/09d8a606-1065-41ce-8052-6df792da5bfd)

Reply:
We dont have this date

Reply:
Unless we need to add more params to the query. Its a graphQl query that you created.

Reply:
We don't have this data, let's not show it